hhs_oauth_server/urls.py

```diff
@@ -25,62 +32,103 @@ def robots_txt(request):
+    # connect/userinfo
     re_path(
-        r"^v1/connect/userinfo", openidconnect_userinfo, name="openid_connect_userinfo"
+        r"^v1/connect/userinfo", openidconnect_userinfo_v1, name="openid_connect_userinfo"
     ),
-    path("v1/fhir/metadata", fhir_conformance, name="fhir_conformance_metadata"),
-    path("v1/fhir/", include("apps.fhir.bluebutton.urls")),
+    # dot_ext
     path("v1/o/", include("apps.dot_ext.urls")),
-    path("v1/o/", include("apps.authorization.urls")),
+    # fhir/bluebutton
+    path("v1/fhir/", include("apps.fhir.bluebutton.urls")),
+    # fhir/metadata
+    path("v1/fhir/metadata", fhir_conformance_v1, name="fhir_conformance_metadata"),
+    # openid_config
+    path(
+        "v1/connect/.well-known/openid-configuration", openid_configuration_v1, name="openid-configuration"
+    ),
+    # smart config
+    path("v1/fhir/.well-known/smart-configuration", smart_configuration_v1, name="smart_configuration"),
+]
+
+urlpatterns_v2 = [
+    # accounts
     path("v2/accounts/", include("apps.accounts.v2.urls")),
+    # authorization
+    path("v2/o/", include("apps.authorization.v2.urls")),
+    # connect/userinfo
     re_path(
         r"^v2/connect/userinfo",
-        openidconnect_userinfo,
+        openidconnect_userinfo_v2,
         name="openid_connect_userinfo_v2",
     ),
-    path("v2/fhir/.well-known/smart-configuration", smart_configuration, name="smart_configuration"),
-    path("v2/fhir/metadata", fhir_conformance_v2, name="fhir_conformance_metadata_v2"),
+    # dot_ext
+    path("v2/o/", include("apps.dot_ext.v2.urls")),
+    # fhir/bluebutton
     path("v2/fhir/", include("apps.fhir.bluebutton.v2.urls")),
+    # fhir/metadata
+    path("v2/fhir/metadata", fhir_conformance_v2, name="fhir_conformance_metadata_v2"),
+    # openid_config
     path(
-        "v3/fhir/.well-known/smart-configuration",
-        waffle_switch("v3_endpoints")(smart_configuration_v3),
-        name="smart_configuration_v3"
+        "v2/connect/.well-known/openid-configuration", openid_configuration_v2, name="openid-configuration-v2"
     ),
-    path("v3/fhir/metadata", waffle_switch("v3_endpoints")(fhir_conformance_v3), name="fhir_conformance_metadata_v3"),
-    path("v3/fhir/", waffle_switch("v3_endpoints")(include("apps.fhir.bluebutton.v3.urls"))),
-    path("v2/o/", include("apps.dot_ext.v2.urls")),
-    path("v2/o/", include("apps.authorization.v2.urls")),
-    path("v3/o/", include("apps.dot_ext.v3.urls")),
+    # smart config
+    path("v2/fhir/.well-known/smart-configuration", smart_configuration_v2, name="smart_configuration"),
+]
+
+urlpatterns_v3 = [
+    # accounts
+    # TODO: This does not exist yet.
+    # path("v3/accounts/", waffle_switch("v3_endpoints")(include("apps.accounts.v3.urls"))),
+    # authorization
     path("v3/o/", include("apps.authorization.v3.urls")),
+    # dot_ext
+    path("v3/o/", include("apps.dot_ext.v3.urls")),
+]
+
+urlpatterns_v3 = urlpatterns_v3 + [
```

Comment:
Given that we removed that if statement you once had in this area, is there any remaining reason to add to the v3 patterns with this instead of having it as part of the initial assignment?